config/packages/monolog.yaml

```diff
@@ -0,0 +1,53 @@
+monolog:
+    channels:
+        - deprecation # Deprecations are logged in the dedicated "deprecation" channel when it exists
+
+when@dev:
+    monolog:
+        handlers:
+            main:
+                type: stream
+                path: "%kernel.logs_dir%/%kernel.environment%.log"
+                level: debug
+                channels: ["!event"]
+            console:
+                type: console
+                process_psr_3_messages: false
+                channels: ["!event", "!doctrine", "!console"]
+
+when@test:
+    monolog:
+        handlers:
+            main:
+                type: fingers_crossed
+                action_level: error
+                handler: nested
+                excluded_http_codes: [404, 405]
+                channels: ["!event"]
+            nested:
+                type: stream
+                path: "%kernel.logs_dir%/%kernel.environment%.log"
+                level: debug
+
+when@prod:
```

Comment:
Est-ce qu'il ne faut pas utiliser un `rotating_file` ? https://symfony.com/doc/current/logging.html#how-to-rotate-your-log-files

Reply:
Ca pourrait être une bonne idée oui, après, j'aimerais bien tester avec la config par défaut pour voir comment ça se comporte et comment on peut l'optimiser.

Reply:
Pas d'objection